hazelcast/src/main/java/com/hazelcast/config/CacheSimpleConfig.java

```diff
@@ -33,7 +39,7 @@
  * CacheConfig depends on the JCache API. If the JCache API is not in the classpath,
  * you can use CacheSimpleConfig as a communicator between the code and CacheConfig.
  */
-public class CacheSimpleConfig {
+public class CacheSimpleConfig implements IdentifiedDataSerializable {
```

Comment:
`CacheConfig` has a dependency on JCache. I want to avoid it. 

Also this feature is adding a dynamic nature to the current static `Config`. 
The current `Config` uses the `CacheSimpleConfig` -> I am using it too. 

Do you see any advantage of `CacheConfig` ?